src/client/App.js

```diff
@@ -1,15 +1,102 @@
-import React from 'react';
+import React, { Component } from 'react';
 
-import ReactImage from './react.png';
 import Header from './components/Header';
+import SearchBar from './components/SearchBar';
+import ProductsContainer from './components/ProductsContainer';
+import Footer from './components/Footer';
 
-const App = () => (
-  <div>
-    <Header />
-    <h1>Hello, world</h1>
-    <h2>Loading... please wait!</h2>
-    <img src={ReactImage} alt="react" />
-  </div>
-);
+class App extends Component {
+  constructor(props) {
```

Comment:
Do not use constructor if you don't need access to props. Just set:
state = {...